requirements.txt

```
@@ -1,3 +1,3 @@
 strawberryfields>=0.15
-pennylane>=0.15
+pennylane>=0.16
```

@rmoyard we don't need to pin here (that was only done in the Cirq plugin).
```suggestion
pennylane>=0.15
```